EIPS/eip-1690.md

```diff
@@ -0,0 +1,174 @@
+---
+eip: 1690
+title: Contract Mortability Standard
+author: Felix Franz (@Felix-Franz)
+discussions-to: https://github.com/ethereum/EIPs/issues/1690
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-01-11
+requires: 173
+---
+
+## Simple Summary
+
+A standard that allows the owner of a contract to completely remove it.
+
+## Abstract
+
+For removing a smart contract the solidity function `selfdestruct(address);` needs to be called from inside the Contract.
+This function allowes to provide a address where the ether currently owned by the smart contract will be transferred before deleting the contract.
+It should be called from outside the contract by the owner only using a public function.
+
+## Motivation
+
+To completely remove a smart contract that is e.g. not needed anymore or was created by mistake, the functionality needs to be implemented in the contract.
+This EIP should standardize this functionality.
+
+## Specification
+
+This standard uses the EIP-173 Contract Ownership Standard to handle the owner management.
+It provides two functions to kill the Smart Contract.
+Both functions must only be executed by the owner.
+One of the function allows to send the contract's ether to any provided address, the other should just transfer the ether to the owner.
```

Comment:
The specification seems to be missing here.  The definition of the two kill functions should be here, and this content should live elsewhere.